doc/workflows_and_tools/enrollment_tools/yubikey_enrollment_tools.rst

```diff
@@ -0,0 +1,136 @@
+.. _yubikey_enrollment_tools:
+
+Yubikey Enrollment Tools
+------------------------
+
+.. index:: Yubikey, Yubico AES mode, Yubikey OATH-HOTP mode
+
+The Yubikey can be used with privacyIDEA in Yubico's own AES mode (*Yubico OTP*),
+in the HOTP mode (*OATH-HOTP*) or the seldomly used static password mode.
+
+This section describes several tools which can be used to initialize and enroll a
+Yubikey with privacyIDEA.
+
+If not using the :ref:`yubico_token` mode, the Yubikey has to be initialized with privacyIDEA
+to bring the token secrets to the system.
+
+privacyIDEA ships several tools to (mass-)enroll Yubikeys in AES mode (Yubikey Token) or HOTP mode (HOTP Token).
+
+.. _privacyideaadm_enrollment:
+
+.. index:: privacyideaadm, admin tool
+
+privacyidea CLI tool
+~~~~~~~~~~~~~~~~~~~~
+
+For Linux Clients, there is the ``privacyidea`` command line
+client [#privacyideaadm]_, to initialize the Yubikeys. You can use the mass enrollment, which
+eases the process of initializing a whole bunch of tokens.
+
+Run the command like this::
+
+   privacyidea -U https://your.privacyidea.server -a admin token \
+   yubikey_mass_enroll --yubimode YUBICO
+
+This command initializes the token and stores the AES secret and prefix
+in privacyIDEA, so the token is immediately useful. You can enroll Yubikeys
+in HOTP mode by removing the option ``--yubimode``.
+You can choose the slot with ``--yubislot``. For further help call
+``privacyidea yubikey_mass_enroll`` with the ``--help`` option and refer to
+the documentation of the tool [#privacyideaadmdocs]_.
+
+You can also use ``yubikey_mass_enroll`` with the option ``--filename`` to
+write the token configuration to the specified file, which can be imported
+later via the privacyIDEA WebUI at Select Tokens -> Import Tokens.
+There, select :ref:`import_oath_csv` and the file you just created.
+
+.. _ykpersgui:
+
+.. index:: Yubikey, Yubikey personalization GUI, Yubikey personalization tool
+
+Yubikey Personalization GUI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also initialize the Yubikey with the official Yubico personalization GUI
+[#ykpers]_ and use the obtained secret to enroll the Yubikey with privacyIDEA.
+For both AES (Yubico OTP) and OATH-HOTP mode, there are two possibilities to initialize
+the Yubikey with privacyIDEA.
+
+Single token enrollment
+=======================
+
+To initialize a single Yubikey in AES mode (Yubico OTP) use the *Quick* button and
+copy the shown secret labeled with "Secret Key (16 bytes Hex)" to the field *OTP Key*
+on the enrollment form in the privacyIDEA WebUI.
+
+.. figure:: images/ykpers-quick-initialize-aes.png
+   :width: 500
+
+   *Initialize a Yubikey in AES mode (Yubikey OTP)*
+
+.. figure:: images/enroll_yubikey.png
+   :width: 500
+
+   *Enroll a Yubikey AES mode token in privacyIDEA*
+
+In the field "Test Yubikey" push the Yubikey button. This will determine the
```

Comment:
The understanding of the Yubikey button is difficult for some users.
I think the best word to be used (I think Yubico uses it themselves) is "touch the Yubikey button".